core/src/com/po/fuck/Assets.java

```diff
@@ -0,0 +1,42 @@
+package com.po.fuck;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.assets.AssetManager;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.utils.JsonReader;
+import com.badlogic.gdx.utils.JsonValue;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Assets {
+
+    public static AssetManager manager = new AssetManager();
+
+    private static final String[] basicAssetsNames = new String[]{"laserBeam", "player", "island",
+            "coin", "bullet", "enemy"};
+    private static final String[] weaponAssetsNames = new String[]{"glock", "laserGun"};
```

Comment:
I understand that this can be used for the initial game loading, but is is though ?
Also this split should not be done here